src/constants.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+export const IDP_POD_INRUPT = "https://broker.pod.inrupt.com";
+export const IDP_POD_COMPAT = "https://broker.pod-compat.inrupt.com";
```

Comment:
I consider these Prod instances (in the same way as Dev, Demo, Staging, etc.), so I'd name them accordingly. I'm a bit on the fence about 'COMPAT' - I'm tempted to refer to that as being WAC-Only, i.e. `IDENTITY_PROVIDER_INRUPT_PROD_WAC_ONLY`...? I'll leave it up to you, but regardless a little comment is warranted to explain the difference.
```suggestion
export const IDENTITY_PROVIDER_INRUPT_PROD = "https://broker.pod.inrupt.com";
export const IDENTITY_PROVIDER_INRUPT_PROD_COMPAT = "https://broker.pod-compat.inrupt.com";
```
Also, I like the idea of these constants being in a vocab somehwere that could be shared internally and externally maybe (i.e. a shared vocab to answer '**_Where are all the official Inrupt Solid servers, and their Brokers, and whatever other services Inrupt might provide (both internal and external)?_**" - but that's obviously a separate ticket!

Reply:
I think that would be great, however to be pedantic I'd call that a knowledge base (or knowledge graph) rather than a vocab :)